test/e2eTests/i18n.js

```diff
@@ -15,30 +14,36 @@ var path = require('path');
  */
 function I18n(options) {
   this.options = util._extend(util._extend({}, this.defaults), options);
-  this.loadTranslations();
+  this.loadTranslations(this.options.locale);
 }
 
 /**
  * Load translations from directory.
  *
  * @api private
  */
-I18n.prototype.loadTranslations = function () {
+I18n.prototype.loadTranslations = function (locale) {
   var self = this;
   var translations = {};
   var llTranslations = {};
-  var locale;
+
+  // Get the subdirectories from the base directory
   var dirs = fs.readdirSync(this.options.directory);
+
   for (var i in dirs) {
     var name = this.options.directory + dirs[i];
 
+    // Each subdirectory represents a module and has it's own locale files
     if (fs.statSync(name).isDirectory()) {
       var files = fs.readdirSync(name);
+
       files.forEach((file) => {
 
         try {
-          locale = path.basename(file).split('.').shift();
-          if (locale.includes("pt")) {
+          fileName = path.basename(file).split('.').shift();
```

Comment:
`localeInFilename`, or something similar, might be a better variable name. `Filename` makes it sound  like it's the full file name.